README.md

```diff
@@ -71,6 +71,17 @@ gradle build
 gradle run -DmainClass=com.google.swarm.tokenization.DLPTextToBigQueryStreamingV2 -Pargs="--region=<region> --project=<project_id> --streaming --enableStreamingEngine --tempLocation=gs://<bucket>/temp --numWorkers=1 --maxNumWorkers=2 --runner=DataflowRunner --filePattern=s3://<bucket>>/file.csv --dataset=<name>  --inspectTemplateName=<inspect_template> --deidentifyTemplateName=<deid_tmplate> --awsRegion=<aws_region> --awsCredentialsProvider=$AWS_CRED"
 ```
 
+## DeIdentification of jsonl files
+JSONL file format uses newline characters to separate JSON values. A sample file can be found at /src/test/resources/sample_data.jsonl
+```
+// Copy the sample jsonl file to GCS
+gsutil cp ./src/test/resources/sample_data.jsonl gs://<bucket>/
+
+// Run the pipeline using following command
+gradle run -DmainClass=com.google.swarm.tokenization.DLPTextToBigQueryStreamingV2 -Pargs=" --region=<region> --project=<projct_id> --streaming --enableStreamingEngine --tempLocation=gs://<bucket>/temp --numWorkers=1 --maxNumWorkers=2 --runner=DataflowRunner --filePattern=gs://<path>.jsonl --dataset=<name>   --inspectTemplateName=<inspect_template> --deidentifyTemplateName=<deid_tmplate> --DLPMethod=DEID --headers=<headers list>"
```

Comment:
Do the headers need to be comma separated? May be in the documentation above, we should call that out.